stdlib/Dates/src/io.jl

```diff
@@ -393,9 +393,48 @@ macro dateformat_str(str)
 end
 
 # Standard formats
+
+#ISO8601 'DateTime' Format for 'date' and 'time' formatting. Default Format.
+#Example
+```jldoctest
+
+julia> Dates.ISODateTimeFormat
+dateformat"yyyy-mm-ddTHH:MM:SS.s"
+
+julia> Dates.format(Dates.now(), Dates.ISODateFormat)
+"2019-04-15T17:18:05.439"
+```
+
```

Comment:
No extra blank line here

Reply:
In addition, you need to include it in a string.